ISISScientificComputing/autoreduce
# 764 fix output structure

### Summary of work
 - Renamed the directory variables in `post_process_admin` so it is more clear that one is temporary and the other final
 - Removed a duplicated log path creation in `post_process_admin`
 - Flat outputs no longer have the number folders
 - Non flat outputs instead have `/run-version-0` and increment on each new version
 - Renamed excitation_instruments to flat_output_instruments
 - Fixes a test that fails only on windows

I have opened this related issue #801

****Note:**** There will be a discrepancy between the real version and the version folder created until the database is cleared and the output folders are cleared.


Previous logging:
![image](https://user-images.githubusercontent.com/44777678/95350603-3e782d00-08b8-11eb-9c33-3be2b198a2ed.png)


New logging:
![image](https://user-images.githubusercontent.com/44777678/95350566-33250180-08b8-11eb-856e-d436fe61408f.png)




### How to test your work
Submit a run for a flat output instrument. Check that it is outputted without the numbered subfolder
Submit a run for a non flat output instrument. Check it is outputted to `run-version-x`
Submit another run and make sure it is incremented

### Additional comments
 

closes #767 closes #764 closes #765


**Before merging ensure the release notes have been updated**
